programmers/난이도별/level01.2016년/HyeonJeong.py

```diff
@@ -1,44 +1,24 @@
 def solution(a, b):
-    answer = ''
-    day_list = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]
-    i = b % 7
-
-    if ( a == 1 or a == 4 or a == 7):
-        if(4 <=i <= 6):
-            answer = day_list [i-4]
-        else:
-            answer = day_list [i+3]
 
-    elif (a == 2 or a == 8):
-        if(i == 0):
-            answer = day_list [6]
-        else:
-            answer = day_list [i-1]
-
-    elif (a == 3 or a == 11):
-            answer = day_list [i]
+    day = b
+    answer = ''
+    total = 0
```

Comment:
```suggestion
    total = b
```
`day`와 `answer`는 굳이 변수로 정의하지 않아도 되지 않을까요?